ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -895,7 +930,7 @@ FEDataManager::prolongData(const int f_data_idx,
     TBOX_ASSERT(n_vars == NDIM); // specialized to side-centered data
     const DofMap& F_dof_map = F_system.get_dof_map();
     SystemDofMapCache& F_dof_map_cache = *getDofMapCache(system_name);
-    System& X_system = d_es->get_system(system_name);
+    System& X_system = d_es->get_system(COORDINATES_SYSTEM_NAME);
```

Comment:
No wonder this never seemed to work properly.........

Reply:
If this is an actual bug, would you mind moving it to a separate PR so we can get it in asap?